data/STRchive-database.json

```diff
@@ -3855,10 +3705,98 @@
         "tr_gnomad":[
             "TR51895"
         ],
-        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset (PMID: 38876750); Adult-onset cortical tremor with epilepsy (PMID: 39349043)",
+        "disease_description":"Cortical tremor, seizures with generalised motor (tonic-clonic) onset [@pmid:38876750]; Adult-onset cortical tremor with epilepsy [@pmid:39349043].",
         "locus_tags":[],
         "disease_tags":[]
     },
+    {
+
+   "chrom":"chr15",
+       "start_hg38":156591766,
+       "stop_hg38":156591783,
+       "start_hg19":156561558,
+       "stop_hg19":156561575,
+       "start_t2t":155728132,
+       "stop_t2t":155728149,
+       "id":"NME_NAXE",
+       "disease_id":"NME",
+       "gene_strand":"+",
+       "reference_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_reference_orientation":[
+           "GGGCC"
+       ],
+       "pathogenic_motif_gene_orientation":[
+           "GGGCC"
+       ],
+       "benign_motif_reference_orientation":[],
+       "benign_motif_gene_orientation":[],
+       "unknown_motif_reference_orientation":[],
+       "unknown_motif_gene_orientation":[],
+       "disease":"NAXE-related mitochondrial encephalopathy",
+       "gene":"NAXE",
+       "flank_motif":null,
+       "locus_structure":"(GGGCC)*",
+       "inheritance":[
+           "AR"
+       ],
+       "type":"5' UTR",
+       "location_in_gene":null,
+       "benign_min":2.0,
+       "benign_max":7.0,
+       "intermediate_min":null,
+       "intermediate_max":null,
+       "pathogenic_min":200.0,
+       "pathogenic_max":200.0,
+       "ref_copies":null,
+       "motif_len":5,
+       "age_onset":"Predominately infantile onset, extending to 20y or older for general disease; proband onset at one year [@doi:10.1038/s41525-024-00429-5].",
+       "age_onset_min":1,
+       "age_onset_max":1,
+       "typ_age_onset_min":0,
+       "typ_age_onset_max":0,
+       "novel":"ref",
+       "mechanism":"LoF",
+       "mechanism_detail":"Reduced NAXE expression from expansion in promoter [@doi:10.1038/s41525-024-00429-5]",
+       "source":[],
+       "details":"Normal range (2-7) alleles established by 484 control alleles and validated with orthogonal databases, while proband had expansion of ~200 repeats [@doi:10.1038/s41525-024-00429-5].",
+       "omim":[
+           "617186"
+       ],
+       "prevalence":null,
+       "prevalence_details":"Proband found in Japanese cohort [@doi:10.1038/s41525-024-00429-5].",
+       "stripy":[],
+       "gnomad":[],
+       "genereviews":[],
+       "mondo":[
+           "0020781"
+       ],
+       "year":"2024 [@doi:10.1038/s41525-024-00429-5]",
```

Comment:
Did you discuss adding citations to year with @vincerubinetti? Just want to make sure we're not going to mess up parsing down the track. This would have flustered my old gene page scripts, but might be fine with the new website build.

Reply:
I'm giving him a list of the fields that can have free text citations. we should be okay